pilot/helpers/agents/Developer.py

```diff
@@ -405,9 +396,12 @@ def execute_task(self, convo, task_description, task_steps, test_command=None, r
                         # if need_to_see_output and 'cli_response' in result:
                         #     result['user_input'] = result['cli_response']
 
-                    elif step['type'] == 'code_change':
+                    elif step['type'] in ['save_file', 'code_change']:
```

Comment:
It's proobably safe to have this as `== 'save_file'` but I'm not sure if this *could* get called as a result of debugging and still use the `code_change` name, so I allow for both.